android_mobile/src/main/java/com/alexstyl/specialdates/donate/DonateActivity.java

```diff
@@ -81,13 +83,13 @@ public void onClick(View v) {
             @Override
             public void run() {
                 scrollToDonate();
-
             }
 
             private void scrollToDonate() {
                 CoordinatorLayout.LayoutParams params = (CoordinatorLayout.LayoutParams) appBarLayout.getLayoutParams();
                 AppBarLayout.Behavior behavior = (AppBarLayout.Behavior) params.getBehavior();
-                behavior.onNestedFling(coordinator, appBarLayout, null, 0, 50, true);
+                if (behavior != null)
```

Comment:
And that brings me to my next comment. In android emulator with google apis installed `hasPlayStoreInstalled()` always returns true so DonationActivity is always launched leading to the following error dominos. So I don't see a better way to handle this for now (if you wanna handle this at all) so it's up to you. It's just a null check that makes the app crash-proof on Android emulators.